cql3/expr/restrictions.cc

```diff
@@ -184,6 +184,7 @@ binary_operator validate_and_prepare_new_restriction(const binary_operator& rest
 
     // Prepare the restriction
     binary_operator prepared_binop = prepare_binary_operator(restriction, db, *schema);
+    expr::verify_no_aggregate_functions(prepared_binop, "where clause");
```

Comment:
Ok, I didn't know that there was an existing error message with that string.
Sad that dtest, instead of detecting bugs (in this case, English bugs), forces us to keep them unfixed :-(